File: frontend/src/app/dtos/customer.ts
export class Customer {


  constructor(private id?: string, private name?: string, private address?: string) {
  }


  get Id(): string {
    return this.id;
  }

  get Name(): string {
    return this.name;
  }
  get Address(): string {
    return this.address;
  }


  set Id(value: string) {
    this.id = value;
  }
  set Name(value: string) {
    this.name = value;
  }
  set Address(value: string) {
    this.address = value;
  }
}

File: demo/src/main/java/com/example/demo/Entity/Customer.java
package com.example.demo.Entity;

import org.springframework.context.annotation.Lazy;

import javax.persistence.*;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;


@Entity
public class Customer  {

    @Id
    private String id;
    private String name;
    private String address;

    @OneToMany(mappedBy = "customer",cascade = {CascadeType.ALL},fetch = FetchType.LAZY)
    private List<Orders> orders=new ArrayList<>();

    public Customer() {
    }

    public Customer(String id, String name, String address) {
        this.setId(id);
        this.setName(name);
        this.setAddress(address);
    }


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    @Override
    public String toString() {
        return "Customer{" +
                "id='" + id + '\'' +
                ", name='" + name + '\'' +
                ", address='" + address + '\'' +
                '}';
    }
}
